tests/commit_log.rs

```diff
@@ -15,12 +16,12 @@ async fn kafka_rest_scoped_subscribe() {
     let server = server::http(move |mut req| async move {
         assert_eq!(req.uri(), "/consumers/farmo-integrator");
 
-        let mut req_body: Vec<u8> = Vec::new();
+        let mut req_body = Vec::new();
         while let Some(item) = req.body_mut().next().await {
             req_body.extend(&*item.unwrap());
         }
 
-        let req_body: Consumer = serde_json::from_slice(&req_body).unwrap();
+        let req_body = serde_json::from_slice::<Consumer>(&req_body).unwrap();
```

Comment:
Is this change (using turbofish instead of type annotations) just a preference or is there some reasoning or common convention behind this approach? I 'tend' to use type annotations when I'm writing rust but I haven't spent much time thinking about which is best.